demo/meeting/src/containers/DynamicMeetingControls/index.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import {
+  ControlBar,
+  AudioInputControl,
+  VideoInputControl,
+  ContentShareControl,
+  AudioOutputControl,
+  ControlBarButton,
+  useUserActivityState,
+  Dots,
+  useDevicePermissionStatus,
+  DevicePermissionStatus,
+} from 'amazon-chime-sdk-component-library-react';
+
+import EndMeetingControl from '../EndMeetingControl';
+import { useNavigation } from '../../providers/NavigationProvider';
+import { StyledControls } from './Styled';
+import DevicePermissionControl from '../DevicePermissionControl/DevicePermissionControl';
+import { DeviceLabels } from '../../types';
+
+const DynamicMeetingControls = () => {
+  const { toggleNavbar, closeRoster, showRoster } = useNavigation();
+  const { isUserActive } = useUserActivityState();
+  const permission = useDevicePermissionStatus();
+
+  const handleToggle = () => {
+    if (showRoster) {
+      closeRoster();
+    }
```

Comment:
I can't image a common use case of the reverse behavior now. If there is any you could see, please let me know, I would love to improve it. This component is only in meeting demo now, the main purpose here is to build a view-only experience and allow QA to test this change.
The requirements for the correct use of this component are a bit strict - it works only if the device permission is unset. In any other cases, the it can't change. This is due to a limitation in our SDK, it would be great we could have a Provider to reveal the device permission status, but it's hard to do this now I think.